src/brogue/Globals.c

```diff
@@ -2450,7 +2450,7 @@ itemTable staffTable[NUMBER_STAFF_KINDS] = {
     {"lightning",       itemWoods[0], "",   15, 1300,   BOLT_LIGHTNING,     {2,4,1}, false, false, "This staff conjures forth deadly arcs of electricity to damage to any number of creatures in a straight line."},
     {"firebolt",        itemWoods[1], "",   15, 1300,   BOLT_FIRE,          {2,4,1}, false, false, "This staff unleashes bursts of magical fire. It will ignite flammable terrain and burn any creature that it hits. Creatures with an immunity to fire will be unaffected by the bolt."},
     {"poison",          itemWoods[3], "",   10, 1200,   BOLT_POISON,        {2,4,1}, false, false, "The vile blast of this twisted staff will imbue its target with a deadly venom. Each turn, a creature that is poisoned will suffer one point of damage per dose of poison it has received, and poisoned creatures will not regenerate lost health until the poison clears."},
-    {"tunneling",       itemWoods[4], "",   10, 1000,   BOLT_TUNNELING,     {2,4,1}, false, false, "Bursts of magic from this staff will pass harmlessly through creatures but will reduce obstructions to rubble."},
+    {"tunneling",       itemWoods[4], "",   10, 1000,   BOLT_TUNNELING,     {2,4,1}, false, false, "Bursts of magic from this staff will pass harmlessly through creatures but will reduce to rubble most obstructions, excepting crystal."},
```

Comment:
I think this one might be better left out, it's not 100% unambiguous anyway as there is more than one type of crystal. Maybe just the "most" is good enough?

```suggestion
    {"tunneling",       itemWoods[4], "",   10, 1000,   BOLT_TUNNELING,     {2,4,1}, false, false, "Bursts of magic from this staff will pass harmlessly through creatures but will reduce most obstructions to rubble."},
```

Reply:
I'm open to either one, I think the player's suggestion was to emphasize that crystals won't (typically) be affected.

Reply:
Is there a kind of crystal that won't reflect tunneling bolts?  Both formations and the outer wall will if I recall.

Reply:
There's outer wall, formations, obstruction crystal, shattering crystal, electric crystals, maybe more. (I don't know which are tunnel-able.) Also I believe tunneling does not work on stairway torches, nor vault walls. So it just seems to me like the exception communicates a false completeness.